src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp/PublicAPI.Unshipped.txt

```diff
@@ -42,7 +42,7 @@ Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnap
 Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnapshot.DocumentFilePaths.get -> System.Collections.Generic.IEnumerable<string!>!
 Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnapshot.FilePath.get -> string!
 Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnapshot.GetDocument(string! filePath) -> Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Document.OmniSharpDocumentSnapshot?
-Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnapshot.ProjectWorkspaceState.get -> Microsoft.CodeAnalysis.Razor.ProjectSystem.ProjectWorkspaceState?
+Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project.OmniSharpProjectSnapshot.ProjectWorkspaceState.get -> Microsoft.AspNetCore.Razor.ProjectEngineHost.Serialization.ProjectWorkspaceState?
```

Comment:
This is actually wrong, and was before, but we a exposing a type that doesn't come from this assembly, which is bad for an EA project to do. If we ever change the json file format, this could be a break. Will log a follow up issue.